### DISCUSSION TITLE
download package and file not found

### DISCUSSION THREAD

**Discussion by eeldeen8951** [minio/minio#16759] [Q&A] [answered]
hi 
I am new to minio 
1: i need to import this package 'minio/minio/internal/init' into the project so i need to download it 
2:when i run the server i got this error (file not found (cmd.StorageErr))

thanks for your help

**Comment by harshavardhana** [comment, marked as answer]
You can't use internal packages outside of MinIO they are called infernal for a reason here.

https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface
